cmd/rookflex/cmd/unmount.go

```diff
@@ -53,7 +52,8 @@ func handleUnmount(cmd *cobra.Command, args []string) error {
 	mounter := getMounter()
 
 	// Check if it's a cephfs
-	err = executor.ExecuteCommand(false, "", "df", "--type", cephFS, mountDir)
+	command := exec.Command("df", "--type", cephFS, mountDir)
+	err = command.Run()
```

Comment:
.. so where does the output from this subprocess go now?
Afaics it still goes to flexvolume stdout.

I think we want something like:
```go
command := exec.Command("df", "--type", cephFS, mountDir)
output, ￼err := command.CombinedOutput()
// maybe log(client, fmt.Sprintf("df output: %s", output), true)
```

Reply:
no, `Cmd` captures both stdout and stderr. `CombinedOutput` just writes the output into a byte array

https://golang.org/src/os/exec/exec.go